File: app/src/main/java/com/halsey/grandtheftradios/RadioObjects/Radio.java
package com.halsey.grandtheftradios.RadioObjects;

public class Radio {
    private final String game;
    private final String name;
    private final String url;

    public Radio(String game, String name, String url) {
        this.game = game;
        this.name = name;
        this.url = url;
    }

    public String getName() {
        return this.name;
    }

    public String getFileName() {
        return this.game + "-" + this.name + ".mp3";
    }

    public String getUrl() {
        return this.url;
    }
}

File: app/src/main/java/com/halsey/grandtheftradios/radio_objects/Radio.kt
package com.halsey.grandtheftradios.radio_objects

class Radio(
    @JvmField val game: String,
    @JvmField val name: String,
    @JvmField val url: String,
    @JvmField val iconDrawable: Int
) {

    val fileName: String
        get() = "$game-$name.mp3"
}
